README.md

```diff
@@ -30,16 +62,24 @@ import("foo.json", { type: "json" })
 <script src="foo.wasm" type="webassembly"></script>
 ```
 
+While the browser can check for WebAssembly's magic number. We would like to offer a way for the developer to assert the type of their ressources.
```

Comment:
Namely, WebAssembly validation will fail if this magic number is not correct. We'd assert that the mimetype matches the type before even beginning to run WebAssembly validation.